### ISSUE TITLE
关于YoloV3-SPP代码的img_utils.py文件中的letterbox函数可能的问题

### ISSUE THREAD

**Issue by DaMiBear** [WZMIAOMIAO/deep-learning-for-image-processing#552]
**System information**
* Have I written custom code: Yes
* OS Platform(e.g., window10 or Linux Ubuntu 16.04): Ubuntu20.04 WSL2
* Python version: 3.8.10
* Deep learning framework and version(e.g., Tensorflow2.1 or Pytorch1.3): Pytorch1.11.0
* Use GPU or not: GPU
* CUDA/cuDNN version(if you use GPU): 11.3
* The network you trained(e.g., Resnet34 network): Yolo V3 SPP

**Describe the current behavior**
up好，我发现在img_utils.py文件中的letterbox函数中，当`scale_fill`为`True`时，下面的这一行赋值根据后面的代码的意思应该是`new_unpad[w,h]=new_shape[w,h]`
[        new_unpad = new_shape](https://github.com/WZMIAOMIAO/deep-learning-for-image-processing/blob/0f334c618885ff8a76af008bf2cc455626e54824/pytorch_object_detection/yolov3_spp/build_utils/img_utils.py#L40)
但是按照之前的代码来理解，就是下面这一行，new_shape应该是`[h,w]`，因为`new_unpad`一定是`[w,h]`
[         dw, dh = new_shape[1] - new_unpad[0], new_shape[0] - new_unpad[1]  # wh padding](https://github.com/WZMIAOMIAO/deep-learning-for-image-processing/blob/0f334c618885ff8a76af008bf2cc455626e54824/pytorch_object_detection/yolov3_spp/build_utils/img_utils.py#L34)
**Error info / logs**

那这样参数`new_shape`的宽和高的顺序不就反了吗？

**Comment by WZMIAOMIAO** [comment]
是的，确实反了，稍后修改。
